src/main/java/org/opentripplanner/street/model/vertex/RentalRestrictionExtension.java

```diff
@@ -5,7 +5,7 @@
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
-import org.opentripplanner.routing.vehicle_rental.GeofencingZone;
+import org.opentripplanner.service.vehiclerental.model.GeofencingZone;
```

Comment:
Could you also move this to `org.opentripplanner.service.vehiclerental.model`

Reply:
Ok so the vertices/edges/extensions also belong in the service package?

Reply:
I would say so, you might want to have a specific street package under the model, where to put these. We can put the additional state logic there as well.

Reply:
What does @t2gran think about this idea? You ask him personally :)

Reply:
I've moved the edge and vertex to a newly added `street` package.